examples/BUILD

```diff
@@ -0,0 +1,37 @@
+#
```

Comment:
Does the empty line here serve any purpose? (Otherwise I'd suggest removing it and just starting with the copyright notice.)

Reply:
Do you plan on addressing this, or we can resolve it?